bedrock/firefox/templates/firefox/features/tips/tips.html

```diff
@@ -16,7 +16,10 @@
 {% block body_id %}firefox-tips{% endblock %}
 
 {% block page_css %}
- {{ css_bundle('firefox_tips') }}
+{{ css_bundle('firefox_tips') }}
```

Comment:
Seeing as the page CSS bundle may contain overrides for certaing CSS components, should it come last in the list here?